core/domain/suggestion_services.py

```diff
@@ -255,9 +258,36 @@ def reject_suggestion(suggestion, reviewer_id, review_message):
     mark_review_completed(
         suggestion, suggestion_models.STATUS_REJECTED, reviewer_id)
 
+    thread_id = suggestion.suggestion_id
+    if not constants.ENABLE_GENERALIZED_FEEDBACK_THREADS:
+        thread_id = thread_id[thread_id.find('.') + 1:]
     feedback_services.create_message(
-        suggestion.suggestion_id, reviewer_id,
-        feedback_models.STATUS_CHOICES_IGNORED, None, review_message)
+        thread_id, reviewer_id, feedback_models.STATUS_CHOICES_IGNORED,
+        None, review_message)
+
+
+def get_all_suggestions_that_can_be_reviewed_by_user(user_id):
+    """Returns a list of suggestions which need to be reviewed, in categories
```

Comment:
How do we propagate the suggestions which can be reviewed because the user is an editor?

Reply:
For now this page only displays suggestions as per category. I believe it will be a complicated query to get suggestions for entities where the given user is an editor (or owner, manager, etc for a general case). So for now I think we should keep it to only based on the category.

Reply:
I think the editors/owners/managers will be the first reviewers. I do not expect that we would have anyone to make suggestions initially. Without this, our system will be rendered useless: editors will never be able to accept/reject suggestions, and hence, suggestors' scores will never increase.

Reply:
Hmmm. So for now, the reviews will continue to happen on the feedback tab page of the explorations (which the editors/owners have access to), and the questions tab (which the managers have access to). This page, I had imagined to be a place for the general reviewer to view the suggestions to review (because the general user doesn't have access to the feedback tab or the questions tab). So this view was targeted towards the users with high scores. Initially I agree, it will not be that useful, but in the long run, this would be a very useful feature (assuming reviews continue to happen as they used to in the feedback tab).

Reply:
Ah, right the editors should be able to see the suggestions on the feedback tab. Then, it is fine I think to not to show the suggestions for your explorations in the common suggestion view. As long as there is a place to show them, I'm happy with it.